seedvault-backup-parser.spec

```diff
@@ -0,0 +1,44 @@
+%define debug_package %{nil}
+
+%define mybuildnumber %{?build_number}%{?!build_number:1}
+
+Name:           seedvault-backup-parser
+Version:        2020.10.23
+Release:        %{mybuildnumber}%{?dist}
+Summary:        Parse Seedvault backups
+BuildArch:      noarch
+
+License:        GPLv2+
```

Comment:
I have licensed the script under Apache 2.0, not GPL.

Reply:
It's now resolved.